tests/unit/cloudformation/test_connection.py

```diff
@@ -149,6 +150,11 @@ def test_update_stack_all_args(self):
             'NotificationARNs.member.2': 'arn:notify2',
             'Parameters.member.1.ParameterKey': 'KeyName',
             'Parameters.member.1.ParameterValue': 'myKeyName',
+            'Parameters.member.2.ParameterKey': 'KeyName2',
+            'Parameters.member.2.ParameterValue': '',
```

Comment:
Again, it's been a few months. But I don't think the API  will accept it otherwise.